zmsadmin/js/page/availabilityDay/form/validate.js

```
@@ -1,34 +1,143 @@
+            errorList.push({
+                type: 'dateOrderInvalid',
+                message: 'Das Startdatum darf nicht nach dem Enddatum liegen.'
+            });
+        }
+    }
+
+    return errorList;
+}
+
+function isValidTimestamp(timestamp) {
+    return !isNaN(timestamp) && moment.unix(timestamp).isValid();
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)